btk/draw_blends.py

```diff
@@ -327,6 +329,8 @@ def render_mini_batch(self, blend_list, psf, wcs, survey, noise_seed, extra_data
             psf (list): List of Galsim objects containing the PSF
             wcs (astropy.wcs.WCS): astropy WCS object
             survey (dict): Dictionary containing survey information.
+            seed_minibatch (numpy.random.SeedSequence): Numpy object for generating
+                random seeds (for the noise generation).
```

Comment:
The parameter name should reflect on the object type, or encompass it.
You call it `seed_minibatch` so I would expect being able to pass a seed, not a seed sequence.

I have so far no opinion whether it's easier to pass a seed and initialise a seed sequence from it at the beginning of the function or directly pass the seed sequence. But it could also be both e.g.
```python
if isinstance(seed_minibatch, int):
    seed_minibatch = np.random.SeedSequence(seed_minibatch)
```
Does not really help regarding the naming I mentioned at the top though. :)

Reply:
After seeing a bit more of the code, I guess the only decent option is to rename `seed_xxx` with `seedseq_xxx` or `seed_seq_xxx` because a seed sequence is a very different from a seed.

Reply:
I agree, I will do that. I think we should always pass the seed sequence up until the point we actually need the integer seed.